packages/lix-sdk/src/snapshot/create-snapshot.ts

```diff
@@ -20,7 +21,7 @@ export async function createSnapshot(args: {
 		const snapshot = await trx
 			.insertInto("snapshot")
 			.values({
-				content: args.content ?? null,
+				content: args.content ? jsonb(args.content) : null,
```

Comment:
Using the jsonb() functoin shouldn't be needed. the `SerializeJsonbPlugin` in `init-db` should take care of it